CHANGELOG.md

```diff
@@ -4,6 +4,20 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/).
 
+## [0.5.0] - 2024-11-01
+
+### Added
+- **Docker Compose support** with multiple `Dockerfile`s for streamlined multi-container setup.
```

Comment:
Is streamlined the right word?

How about: "improved architecture by running each service in a separate container"